crates/bevy_ui/src/entity.rs

```diff
@@ -39,6 +42,38 @@ pub struct NodeBundle {
     pub computed_visibility: ComputedVisibility,
 }
 
+impl NodeBundle {
+    pub fn new(style: Style, color: impl Into<UiColor>) -> Self {
+        Self {
+            style,
+            color: color.into(),
+            ..default()
+        }
+    }
+
+    /// Returns a node without a background color and that will let interactions go through them
+    /// A transparent node is like a <div> element in HTML
```

Comment:
```suggestion
    /// A container node is like a <div> element in HTML
```

Make the documentation match up with the name of the method.

Reply:
Thank you!